src/router/index.js

```diff
@@ -1,22 +1,8 @@
 import { createRouter, createWebHistory } from "vue-router";
-import HomeView from "../views/HomeView.vue";
 
 const router = createRouter({
   history: createWebHistory(import.meta.env.BASE_URL),
   routes: [
-    {
```

Comment:
Please keep HomeView, refactor and imported as your home page.
You can read about RouterLink and how its work in VueJS